docs/getting-started/README.md

```diff
@@ -392,6 +392,7 @@ Simply commit this file to the root directory of your repo. Every subsequent sca
 
 !!! Note
     Only SAST findings are supported in baseline. Support for depscan findings is coming shortly.
+    The baseline file is included as a part of the artifact report generated by Github Actions. The file `.sastscan.baseline` seems to be missing in MacOS but is available in Windows. 
```

Comment:
oh, this sounds like a bug! Could you file it as a bug instead? Is there any public pipeline to test this?